client/state/data-layer/wpcom/sites/media/index.js

```diff
@@ -61,8 +70,15 @@ export function updateMedia( action ) {
 	];
 }
 
-export const updateMediaSuccess = ( { siteId }, mediaItem ) => ( dispatch ) => {
+export const updateMediaSuccess = ( { siteId }, mediaItem ) => ( dispatch, getState ) => {
 	dispatch( receiveMedia( siteId, mediaItem ) );
+
+	if ( isEditorIframeLoaded( getState() ) ) {
+		const iframePort = getEditorIframePort( getState() );
+		const payload = createMediaGutenframePayload( 'updated', mediaItem );
```

Comment:
Since there's a chance we can lack the iframe port (if we're using the classic Calypso editor), should we check if the `iframePort` is in place before using it?

Reply:
done via https://github.com/Automattic/wp-calypso/commit/28ccf690cba9a2f11b85a9e43c27e31df60d9045